src/main.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { render as reactDomRender } from 'react-dom';
+import superagent from 'superagent';
+import '../style/main.scss';
+
+// const apiURL = `https://reddit.com/r/${searchFormBoard}.json?limit=${searchFormLimit}`;
+// TODO use the above line for final lab
+
+class RedditSearchForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      redditBoardName: '',
+      returnResults: 0,
+    };
+
+    this.handleBoardChange = this.handleBoardChange.bind(this);
+    this.handleBoardNumberChange = this.handleBoardNumberChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleSubmitNumber = this.handleSubmitNumber.bind(this);
+  }
+
+
+  handleBoardChange(event) {
+    this.setState({ redditBoardName: event.target.value });
+  }
+
+  handleBoardNumberChange(event) {
+    this.setState({ returnResults: event.target.value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    this.props.redditBoardSelect(this.state.redditBoardName);
```

Comment:
pass in two arguments into `redditBoardSelect`, one with the board name and one with the search limit